### PR TITLE
Fix inconsistencies with schematics

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: src/mainboard/clevo/mtl-h/devicetree.cb [hunk 1]
```diff
@@ -26,61 +26,28 @@ chip soc/intel/meteorlake
 	register "pch_pm_energy_report_enable" = "1"
 
 	# Thermal
-	register "tcc_offset" = "30" # TCC of 100C
+	register "tcc_offset" = "30" # TCC of 80C
 
-	# Power limits
+	# Power limits, why PL1 = 35W if TDP is 28W?
```
Comment (mkopec) [inline src/mainboard/clevo/mtl-h/devicetree.cb:31]: I bumped it a bit to improve performance, because the cooling system can handle it

Comment (miczyg1) [reply src/mainboard/clevo/mtl-h/devicetree.cb:31]: Then why we hit overheating issues?

Comment (mkopec) [reply src/mainboard/clevo/mtl-h/devicetree.cb:31]: I don't know yet. Did rolling it back to 28W fix that issue?

Comment (miczyg1) [reply src/mainboard/clevo/mtl-h/devicetree.cb:31]: Didn't test that yet, thus draft. For now looking for things that may be wrong or affect the heat generation

Comment (miczyg1) [reply src/mainboard/clevo/mtl-h/devicetree.cb]: Reverted back to 35W and found possible culprit of overheating: https://github.com/Dasharo/ec/pull/62#discussion_r1750048659

Also added PsysPmax setting to coreboot.

File: src/mainboard/clevo/mtl-h/devicetree.cb [hunk 2]
```diff
@@ -208,38 +210,17 @@ chip soc/intel/meteorlake
...
 			register "serial_io_i2c_mode[PchSerialIoIndexI2C3]" = "PchSerialIoPci"
 		end
-		device ref pcie_rp5 on # GLAN
-			register "pcie_rp[PCH_RP(5)]" = "{
-				.clk_src = 2,
-				.clk_req = 2,
-				.flags = PCIE_RP_LTR | PCIE_RP_CLK_SRC_UNUSED,
-			}"
-			register "pcie_clk_config_flag[2]" = "PCIE_CLK_LAN"
+		device ref i2c4 on # ANX7443 USB-C Retimer
```
Comment (mkopec) [inline src/mainboard/clevo/mtl-h/devicetree.cb:219]: Only on dGPU models, please move to overridetree (if it's not already there)

File: src/mainboard/clevo/mtl-h/devicetree.cb [hunk 3]
```diff
@@ -208,38 +210,17 @@ chip soc/intel/meteorlake
...
+			register "serial_io_i2c_mode[PchSerialIoIndexI2C4]" = "PchSerialIoPci"
+		end
+		device ref i2c5 on # Smart AMP
```
Comment (mkopec) [inline src/mainboard/clevo/mtl-h/devicetree.cb:222]: Only on 16" dGPU models, already defined in dGPU overridetree

Comment (miczyg1) [reply src/mainboard/clevo/mtl-h/devicetree.cb:222]: It is also optional on iGPU models per schematics (at least V560TU).

File: src/mainboard/clevo/mtl-h/devicetree.cb (hunk 2 above)
Comment (miczyg1) [reply src/mainboard/clevo/mtl-h/devicetree.cb:219]: Right, my bad

File: src/mainboard/clevo/mtl-h/variants/dgpu/gpio.c [hunk 4]
```diff
@@ -212,39 +212,39 @@ static const struct pad_config gpio_table[] = {
 	/* ------- GPIO Community 5 ------- */
 
 	/* ------- GPIO Group GPP_B ------- */
-	PAD_CFG_GPO(GPP_B00, 0, DEEP),	/* GPIO */
+	PAD_CFG_GPO(GPP_B00, 0, DEEP),	/* GPIO - TP_ATTN# (alternative?) */
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply src/mainboard/clevo/mtl-h/variants/dgpu/gpio.c]: https://github.com/Dasharo/coreboot/pull/557/commits/ce4e8e305b2aaa4220583324be7d323b800d48cf#diff-524ed62d469860f4c4a5d05f059d42ce75767de0d8d9fea0651a8a5dcbde9d82R215

File: src/mainboard/clevo/mtl-h/devicetree.cb (hunk 2 above)
Comment (miczyg1) [reply src/mainboard/clevo/mtl-h/devicetree.cb]: Removed: https://github.com/Dasharo/coreboot/pull/557/commits/ce4e8e305b2aaa4220583324be7d323b800d48cf#diff-3d93912120f7f4c466fd40100f6e49ad0e915222a34652442b4ec95349947ecfR218

**Review by mkopec — APPROVED** [review]
Looks good now, thanks

**Pull request merged**